src/Sentry/TracePropagationTarget.cs

```diff
@@ -1,60 +0,0 @@
-namespace Sentry;
-
-// TODO: Replace TracePropagationTarget with SubstringOrRegexPattern in next major version.
```

Comment:
It doesn't really explain why.
This type at least from what its named is part of Sentry's protocol and easy to find in docs, etc.

It's getting removed just because it's 'the same as SubstringOrRegexPattern'?

Reply:
I put some context in the PR description. I think the `TracePropagationTargets` property will still be easy enough to find and is well documented and having it's own type wouldn't make it any more usable:
https://github.com/getsentry/sentry-dotnet/blob/f2524bdf758d3af0192e7ba91d24abb21a1a562d/src/Sentry/SentryOptions.cs#L824-L842